### DISCUSSION TITLE
[漏洞]

### DISCUSSION THREAD

**Discussion by Huchangzhi** [win12-online/win12#324] [Q&A] [answered]
## 描述

一直在加载页面

## 重现

按以下步骤可重新复现该漏洞：

灰常简单，在360浏览器里怎么都打不开

## 截图

（如果有的话，请在这里附上相关的截图或视频）

## 测试环境

浏览器：360
浏览器版本：最新版本
操作系统：win11

## 其他

试了好几个电脑，都这样

**Comment by User782Tec** [comment]
一般是网络问题，我这边能正常加载
感谢您的反馈

**Comment by tjy-gitnub** [comment, marked as answer]
根据我这边的测试，是js文件源过慢
暂时的解决方法：
1. 翻墙加速
2. 反复刷新若干次
3. 使用chrome
4. 使用devtool强制开机（删掉#loadback，但是将无法使用一些高端功能
感谢反馈~
